test/unit/spec/v2/ion/uiSchemaTransformer_spec.js

```diff
@@ -1,223 +1,792 @@
-              'name': 'otp',
-              'label': 'Passcode',
-              'minLength': 4,
-              type: 'text',
+              'name':'factorProfileId',
+              'type':'factorType',
+              'options':[
+                {
+                  'label':'Password Label',
+                  'value':'00u2j17ObFUsbGfLg0g4',
+                  'method':'options',
+                  'factorType':'password'
+                },
+                {
+                  'label':'Email Label',
+                  'value':'emf2j1ccd6CF4IWFY0g3',
+                  'method':'options',
+                  'factorType':'email'
+                }
+              ],
+              'method':'post'
             }
-          ],
+          ]
         }
       ]
     });
   });
 
   it('converts response with fields as form for ENROLL_PROFILE', () => {
-    const result = _.compose(uiSchemaTransformer, actionsTransformer, responseTransformer)(XHREnrollProfile.response);
-    expect(result.currentState).toEqual({
-      'version': '1.0.0',
-      'stateHandle': '01r2p5S9qaAjESMFuPzt7r3ZMcZZQ_vvS0Tzg56ajB',
-      'expiresAt': '2019-07-24T21:25:33.000Z',
-      'step': 'PROFILE_REQUIRED',
-      'intent': 'LOGIN',
-      'enroll-profile': jasmine.any(Function),
-      'select-identify': jasmine.any(Function),
-      'cancel': jasmine.any(Function),
-      'context': jasmine.any(Function),
-      'remediation': [
+    const rawUserEnrollResponse = XHREnrollProfile.response;
+    const transformedResponse  = {
+      '__rawResponse':rawUserEnrollResponse,
+      'neededToProceed':{
+        'enroll-profile':[
+          {
+            'name':'userProfile',
+            'form':{
+              'value':[
+                {
+                  'name':'lastName',
+                  'label':'Last name',
+                  'required':true,
+                  'method':'post'
+                },
+                {
+                  'name':'firstName',
+                  'label':'First name',
+                  'required':true,
+                  'method':'post'
+                },
+                {
+                  'name':'email',
+                  'label':'Primary email',
+                  'required':true,
+                  'method':'post'
+                }
+              ],
+              'method':'post'
+            },
+            'method':'post'
+          }
+        ],
+        'select-identify':[
+          {
+            'name':'identifier',
+            'label':'identifier',
+            'method':'post'
+          }
+        ]
+      },
+      'actions':{
+   
+      },
+      'context':{
+        'stateHandle':'01r2p5S9qaAjESMFuPzt7r3ZMcZZQ_vvS0Tzg56ajB',
+        'version':'1.0.0',
+        'expiresAt':'2019-07-24T21:25:33.000Z',
+        'step':'PROFILE_REQUIRED',
+        'intent':'LOGIN'
+      },
+      'rawIdxState':rawUserEnrollResponse
+    };
+    const result = _.compose(uiSchemaTransformer, responseTransformer)(transformedResponse);
```

Comment:
We seem to be testing both uiSchemaTransformer, responseTransformer here. Better to just test uiSchemaTransformer ?